test/summary.cc

```diff
@@ -95,13 +95,23 @@ TEST(Summary, test_time_summary) {
     for (const auto& leg : route.legs()) {
       // accumulate the maneuvers for a leg
       double accumulated_time = 0;
+      double accumulated_transition_time = 0;
```

Comment:
two types of test additions here. first one is to make sure that transition times are less than maneuver times that encapsulate them and second is to make sure that transition times are non zero, also tests the json output in these ways